indexer/editable_map_object.cpp

```diff
@@ -368,6 +368,12 @@ void EditableMapObject::RemoveBlankAndDuplicationsForDefault()
   m_name = editedName;
 }
 
+void EditableMapObject::RemoveNameForLanguage(int8_t langCode)
+{
+  if (m_name.HasString(langCode))
+    m_name.RemoveString(langCode);
```

Comment:
1. The original feature in OSM has several translations (e.g. name, name:ru, name:en, etc.)
2. Edit Place and delete a translation (e.g. remove name:ru or name:en), save and move OM to the background.
3. Will the name:ru or name:en be deleted in OSM?

---

1. Create a new object and give it several translations. Save. Move OM to background to upload changes.
2. Make sure that object is added to OSM with a proper tags.
3. Edit the same object and delete some translation. Save. Move OM to background.
4. Will removed translation be reflected in OSM?

CC @map-per